configs/train_configs/sft/default.yaml

```diff
@@ -4,7 +4,14 @@ use_flash_attn: true
 tokenizer_name: allenai/OLMo-1.7-1B-hf
 use_slow_tokenizer: false # olmo models only use fast tokenizers
 # train_file: data/processed/tulu_v2/tulu_v2_data.jsonl
-dataset_name: allenai/tulu-v2-sft-mixture
+# dataset_name: allenai/tulu-v2-sft-mixture
+dataset_mixer:
+ allenai/tulu-v2-sft-mixture: 0.5
+ HuggingFaceH4/no_robots: 0.8
+#  data/processed/tulu_v2/tulu_v2_filtered_data.jsonl: 0.1
+# dataset_mixer:
+#  allenai/tulu-v2-sft-mixture: 100000
+#  HuggingFaceH4/no_robots: 5000
 max_seq_length: 4096
 max_train_samples: 10000
```

Comment:
nit: this can be a good example of a custom mix in a different config, but the default config should replicate the tulu v2 mix (i.e. just do 100% of the default or olmo version of the v2 mix)

Reply:
the tulu mix is pretty annoying to replicate, so idk if I agree. It should be something easy to run (hence using a 1B model too)

Reply:
oh I just meant calling the standard v2 mix from HF, but that's also fine with me